primer/primer.style
# Bugfix/fix redirects child pages

### Problem:

The fix in #272 restored top-level pages, but not inner pages. This means that https://primer.style/react/system-props (as an example) isn't resolving to the correct downstream route. 

### Solution

Update the redirect parameters to include child routes

### How to test

Similarly to #272, it's tricky to have confidence until we're in production due to absolute urls. 

Testing this on the preview deployment, does work okay however:

1) Go here: https://primer-style-git-bugfix-fix-redirects-child-pages-primer.vercel.app/react
2) Click on a url in the sidebar
3) Verify that the url _and_ the page content has changed to what is expected 

####  Notes

⚠️ The network requests still indicate that a noticeable amount of redirects are still occurring despite the page loading correctly eventually. These will need to be resolved downstream of this repo.
